pkg/validation/options_test.go

```diff
@@ -78,12 +77,19 @@ func TestClientSecretFileOption(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to create temp file: %v", err)
 	}
-	f.WriteString("testcase")
+	_, err = f.WriteString("testcase")
```

Comment:
I don't know why my IDE catches these when I try to commit but GoSec doesn't 🤷‍♂️ 

These are unrelated to this PR, just handling errors that could be thrown.